jcommune-service/src/main/java/org/jtalks/jcommune/service/transactional/TransactionalPostCommentService.java

```diff
@@ -62,6 +62,8 @@ public PostComment updateComment(long id, String body, long branchId) throws Not
         PostComment comment = get(id);
         checkHasUpdatePermission(comment, branchId);
         comment.setBody(body);
+        comment.setUserChanged(userService.getCurrentUser());
+        comment.setModificationDate(new DateTime(System.currentTimeMillis()));
```

Comment:
made same as in this case - comment.setCreationDate(new DateTime(System.currentTimeMillis()));

fixed